### DISCUSSION TITLE
How to draw a curve at the bottom corner of border element

### DISCUSSION THREAD

**Discussion by SubashiniMahendran** [dotnet/wpf#6943] [General]
Hi Team,

My requirement is to draw a curve for the border element in **XAML** as shown in below screenshot:

![image](https://user-images.githubusercontent.com/92976485/185896474-9ec65f15-8915-4a3f-a888-581044fccd65.png)

Can anyone suggest a solution to achieve this UI?

**Comment by miloush** [comment]
There is no easy way to do this. One option would be to use the Path element, see e.g. [Geometry overview](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/graphics-multimedia/geometry-overview?view=netframeworkdesktop-4.8).

**Comment by dipeshmsft** [comment]
Moving to discussion.
